README.md

```diff
@@ -141,6 +144,28 @@ brew install nginx-full --with-tarantool-module
 
 ```
 
+
+[Back to contents](#contents)
+
+## Test run and notes for contributors
+
+This paragraph is actually only if you chose [build from source](#build-from-source)

```

Comment:
Grammar:

1. 'Actually' is adverb, 'actual' is adjective. Use the latter here.
2. 'Chose' is the past tense form, let's use infinitive form 'choose' here.

Reply:
Fixed.